src/execution/execution.cc

```diff
@@ -392,8 +392,11 @@ V8_WARN_UNUSED_RESULT MaybeHandle<Object> Invoke(Isolate* isolate,
         !recordreplay::AreEventsDisallowed() &&
         V8RecordReplayDependencyGraphExecutionNode() == 0 &&
         function->shared().script().IsScript()) {
-      std::string location = GetFunctionLocationInfo(isolate, function);
-      recordreplay::Warning("DependencyGraph missing execution: %s", location.c_str());
+      static bool show_warning = !!getenv("RECORD_REPLAY_WARN_MISSING_EXECUTION");
```

Comment:
Since this is replay-time only, should this not technically have the `REPLAYING_` prefix?

Reply:
The env var still needs to be set while recording even though it only has an effect while replaying.